game/src/sandbox/mod.rs

```diff
@@ -293,6 +296,21 @@ impl State for SandboxMode {
             }
         }
 
+        // We need to recalculate unzoomed agent mouseover when the mouse is still and time passes,
```

Comment:
> when the mouse is still and time passes

Is this because an unsuspecting agent can walk "under" the cursor? That's sneaky. 👀